.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,7 @@
 <!-- Thanks so much for your PR, your contribution is appreciated! ❤️ -->
 
-- [ ] I have followed (at least) the [PR section of the contributing guide](https://github.com/mui/material-ui/blob/HEAD/CONTRIBUTING.md#sending-a-pull-request).
+- Fixes <!-- Issue hyperlink -->
```

Comment:
Should we add checkboxes? We could later introduce a PR check which validates that all boxes are checked.

There were concerns about increased friction, but IMO adding a `x` between `[ ]` should not be such a huge friction

Reply:
I feel like adding the checkbox adds very little value, since I would rather just add the image instead of checking a box

Reply:
I brought it up only because it allows us to use PR checks to validate if action was done, however it's fine if we skip that for now since we don't have checks anyways 👍